docs/user/guides/subprojects.rst

```diff
@@ -14,24 +14,30 @@ Adding a subproject
 In the admin dashboard for your project, select :guilabel:`Subprojects` from the menu.
 From this page you can add a subproject by typing in the project slug.
 
+Immediately after adding the subproject, its default version will be visible from the link displayed in the list of subprojects.
+
 .. note::
 
    * On |org_brand|, you need to be maintainer of a subproject in order to choose it from your main project.
    * On |com_brand|, you need to have admin access to the subproject in order to choose it from your main project.
 
-Subproject aliases
-~~~~~~~~~~~~~~~~~~
+Editing a subproject
+--------------------
+
+You can edit a subproject at any time by clicking :guilabel:`|:memo:|` in the list of subprojects.
+On the following page, it's possible to both change the alias and source project.
+Click :guilabel:`Update subproject` to save your changes.
+
+What is visible in ``/projects/<subproject-alias>/`` will be updated immediately when you save your changes.
+
+
+Deleting a subproject
+---------------------
 
-You can choose an alias for the subproject when it is created.
-This allows you to override the URL that is used to access it,
-giving more control over how you want to structure your projects.
+You can edit a subproject at any time by clicking :guilabel:`|:memo:|` in the list of subprojects.
+On the edit page, click :guilabel:`Delete subproject`.
 
-You can set your subproject's project name and :term:`slug` however you want,
-but we suggest prefixing it with the name of the main project.
+Your subproject will be removed immediately and it will stop redirecting from its own domain.
 
-Typically, a subproject is created with a ``<mainproject>-`` prefix,
-for instance if the main project is called ``foo`` and the subproject is called ``bar``,
-then the subproject's Read the Docs project name will be ``foo-bar``.
-When adding the subproject,
-the alias is set to ``bar`` and the project's URL becomes
-``foo.readthedocs.io/projects/bar``.
+Deleting a subproject does not mean to completely remove that project.
+It is solely about removing the reference from the main project.
```

Comment:
```suggestion
**Deleting a subproject does not mean to completely remove that project**.
It removes the reference from the main project.
```

Perhaps this is useful. I think an admonition is probably too much here, but it's good to highlight.